docs/releasenotes/10.2.0.rst

```diff
@@ -105,3 +107,15 @@ Encoder errors now report error detail as string
 
 :py:exc:`OSError` exceptions from image encoders now include a textual description of
 the error instead of a numeric error code.
+
+Type hints
+^^^^^^^^^^
+
+Work has begun to add type annotations to Pillow, including:
+
+* :py:mod:`~PIL.ContainerIO`
+* :py:mod:`~PIL.FontFile` and subclasses
```

Comment:
```suggestion
* :py:mod:`~PIL.FontFile`, :py:mod:`~PIL.BdfFontFile` and :py:mod:`~PIL.PcfFontFile`
```
If we're linking to the module, which is fair enough, then `BdfFontFile` and `PcfFontFile` aren't subclasses of the module.